src/styl/responsive/mobile.styl

```diff
@@ -140,3 +140,66 @@ font-sizes($font-size-basic, $font-size-extra, $font-size-large, $font-size-mid,
   height: 33vw
   .cube-left
     cursor: pointer
+
+// ******************************************************
+
+// AGE CHECKER FORM
+
+.overlay-position
+  position: fixed
+  width: 100vw
+  height: 100vw
+  top:0
+  left:0
+  z-index: 9999
+
+.container-blur
+  position: absolute
+  width: 100%
+  height: 100%
+  top:0
+  left:0
+  
+.container-overlay
+  position: absolute
+  width: 100vw
+  height: 100vw
```

Comment:
i think better to use `100%` for these values, as the parent div is already `100vw`,`100vh`